jablib/src/main/java/org/jabref/logic/cleanup/ConvertToBiblatexCleanup.java

```diff
@@ -53,6 +54,27 @@ public List<FieldChange> cleanup(BibEntry entry) {
                 }
             });
         }
+        // If still no 'date' field, try fallback logic
+        if (entry.getField(StandardField.DATE).isEmpty()) {
+            changes.addAll(applyDateFallback(entry));
+        }
+
+        return changes;
+    }
+
+    private Collection<? extends FieldChange> applyDateFallback(BibEntry entry) {
```

Comment:
Oh, another thing -- can we use more specificity here? 
```suggestion
    private List<FieldChange> applyDateFallback(BibEntry entry) {
```
Or was there a reason you chose `Collection<? extends FieldChange>`?

(In case you go with this suggestion, remove the unused `Collection` import too).

Reply:
My bad! I used IntelliJ’s autocomplete but didn’t pay enough attention to the return type 😭. Will change it to List<FieldChange>, thanks!